lib/widgets/message_list.dart

```diff
@@ -250,6 +247,30 @@ class _MessageListState extends State<MessageList> with PerAccountStoreAwareStat
       // TODO: Offer `ScrollViewKeyboardDismissBehavior.interactive` (or
       //   similar) if that is ever offered:
       //     https://github.com/flutter/flutter/issues/57609#issuecomment-1355340849
+      findChildIndexCallback: (Key key) {
```

Comment:
This comment is about the code it was immediately next to, namely `keyboardDismissBehavior`. So it should stay attached to that — otherwise the comment starts becoming puzzling to understand, and also someone looking at the relevant code is no longer likely to notice the comment.